app/src/main/java/org/tokend/template/logic/persistance/FingerprintAuthManager.kt

```diff
@@ -17,6 +17,12 @@ class FingerprintAuthManager(
     private var successCallback: ((String, CharArray) -> Unit)? = null
     private var errorCallback: ((String?) -> Unit)? = null
 
+    private val preferences = applicationContext.defaultSharedPreferences
+
+    val isAuthAvailable: Boolean
```

Comment:
In method `requestAuthIfAvailable` the check is different. What's the reason to use this one?